synapse/storage/engines/postgres.py

```diff
@@ -47,6 +60,7 @@ def on_new_connection(self, db_conn):
         if not self.synchronous_commit:
             cursor = db_conn.cursor()
             cursor.execute("SET synchronous_commit TO OFF")
+            cursor.execute("SET bytea_output TO escape")
```

Comment:
This does not look like the right place for it.
`synchronous_commit` defaults to `true` so this will in default not do that what you expect it to do.

Reply:
wait, shit. that's right. thanks for catching it, let me fix that up.